classes/AddonsManager.php

```diff
@@ -40,7 +40,7 @@ public static function register() {
 		/**
 		 * If single course and built with elementor then remove description addon for avoiding the_content overlap
 		 */
-		if ( $post->post_type && $post->post_type == tutor()->course_post_type ) {
+		if ( isset( $post->post_type ) && $post->post_type == tutor()->course_post_type ) {
```

Comment:
Use strict compare `===`

Reply:
Done, but that was not the point of the problem. :)